server/views/pages/view/calculationSummary.njk

```diff
@@ -40,17 +42,47 @@
     {% include "../partials/ersedCannotHappenNotificationBanner.njk" %}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            <h1 class="govuk-heading-xl">Dates {% if model.prisonerDetail %}for {{ model.prisonerDetail.firstName | title + ' ' + model.prisonerDetail.lastName | title }}
-                {% endif %}
-            </h1>
+            <h class="govuk-caption-xl">Look up a calculation</h>
```

Comment:
No heading level number?

Reply:
This should have been <span> from the prototype, I've fixed